src/map/script.c

```diff
@@ -9688,10 +9688,23 @@ BUILDIN(setgroupid) {
 
 /// Returns the group ID of the player.
 ///
-/// getgroupid() -> <int>
+/// getgroupid({<account ID>/<character ID>/<character name>})
```

Comment:
Eh, I can't really say I'm fond of functions that take an ID, magically figure out what kind of ID is is and try to use it, and eventually silently fail if no such player is currently online, without a possibility to detect the condition from the script.